### ISSUE TITLE
[data request] SPIE-AAPM-NCI BreastPathQ

### ISSUE THREAD

**Issue by ziyuan-shen** [tensorflow/datasets#982]
* Name of dataset: SPIE-AAPM-NCI BreastPathQ
* URL of dataset: http://spiechallenges.cloudapp.net/competitions/14#participate
* License of dataset: Unknown
* Short description of dataset and use case(s): The dataset's training/validation set consists of 2578 patches extracted from 96 whole slide images (WSI). Each patch is labelled by a tumor cellularity score. The testing set contains 1121 patches from 25 WSIs. Labels for testing data are not provided by far. The dataset can be used to develop an automated method for evaluating cancer cellularity from histology patches extracted from WSIs. The method is aimed to increase reproducibility of cancer cellularity scores and enhance tumor burden assessment.

Folks who would also like to see this dataset in `tensorflow/datasets`, please thumbs-up so the developers can know which requests to prioritize.

And if you'd like to contribute the dataset (thank you!), see our [guide to adding a dataset](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).

**Comment by ziyuan-shen** [comment]
Working on it
